v2/pkg/protocols/http/operators_test.go

```diff
@@ -168,6 +168,20 @@ func TestHTTPOperatorExtract(t *testing.T) {
 		require.Greater(t, len(data), 0, "could not extractor kval valid response")
 		require.Equal(t, map[string]struct{}{"Test-Response": {}}, data, "could not extract correct kval data")
 	})
+
+	t.Run("json", func(t *testing.T) {
+		extractor := &extractors.Extractor{
+			Type: "json",
+			Json: []string{".batters | .batter | .[] | .id"},
+		}
+		err = extractor.CompileExtractors()
+		require.Nil(t, err, "could not compile json extractor")
+
+		event["body"] = exampleJSONResponseBody
```

Comment:
also not quite sure about this approach

Reply:
the test looks good to me, it performs the intended task